pkg/webhook/configuration.go

```diff
@@ -486,10 +486,79 @@ func addCVCWithUpdateRule(config *v1beta1.ValidatingWebhookConfiguration) {
 	}
 }
 
+func getOldService(openebsNamespace string) (*corev1.ServiceList, error) {
+	v100SVCName := "admission-server-svc"
+	// fetch service 1.1.0 onwards based on label
+	svcList, err := svc.NewKubeClient(svc.WithNamespace(openebsNamespace)).List(metav1.ListOptions{LabelSelector: webhooksvcLabel})
+	if err != nil {
+		return nil, err
+	}
+	// for 1.0.0 fetch service based on old name
+	oldSVC, err := svc.NewKubeClient(svc.WithNamespace(openebsNamespace)).Get(v100SVCName, metav1.GetOptions{})
+	if err != nil && !k8serror.IsNotFound(err) {
+		return nil, err
+	}
+
+	if err == nil {
+		svcList.Items = append(svcList.Items, *oldSVC)
+	}
+
+	return svcList, nil
+}
+
+func getOldSecret(openebsNamespace string) (*corev1.SecretList, error) {
+	v100SecretName := "admission-server-certs"
+	// fetch secret 1.1.0 onwards based on label
+	secretList, err := secret.NewKubeClient(secret.WithNamespace(openebsNamespace)).
+		List(metav1.ListOptions{LabelSelector: webhookLabel})
+	if err != nil {
+		return nil, err
+	}
+	// for 1.0.0 fetch secret based on old name
+	oldSecret, err := secret.NewKubeClient(secret.WithNamespace(openebsNamespace)).
+		Get(v100SecretName, metav1.GetOptions{})
+	if err != nil && !k8serror.IsNotFound(err) {
+		return nil, err
+	}
+	if err == nil {
+		secretList.Items = append(secretList.Items, *oldSecret)
+	}
+	return secretList, nil
+
+}
+
+func getOldConfig() (*v1beta1.ValidatingWebhookConfigurationList, error) {
```

Comment:
How this file got changed ? I think you have to rebase the changes against the master branch.

Reply:
no idea will rebase! :)